climada/entity/entity_def.py

```diff
@@ -69,22 +89,26 @@ def read_mat(self, file_name, description=''):
             one description of the
             data or a description of each data file
 
-        Raises
-        ------
-        ValueError
+        Returns
+        -------
+        ent : climada.entity.Entity
+            The entity from matlab file
         """
-        self.exposures = Exposures.from_mat(file_name)
-
-        self.disc_rates = DiscRates()
-        self.disc_rates.read_mat(file_name, description)
-
-        self.impact_funcs = ImpactFuncSet()
-        self.impact_funcs.read_mat(file_name, description)
-
-        self.measures = MeasureSet()
-        self.measures.read_mat(file_name, description)
-
-    def read_excel(self, file_name, description=''):
+        return cls(
+            exposures = Exposures.from_mat(file_name),
+            disc_rates = DiscRates.from_mat(file_name, description),
+            impact_func_set = ImpactFuncSet.from_mat(file_name, description),
+            measure_set = MeasureSet.from_mat(file_name, description)
```

Comment:
Don't use spaces around the = sign when used to indicate a keyword argument ([PEP 8](https://www.python.org/dev/peps/pep-0008/))